### PR TITLE
Update profile document access control

### LINKED ISSUE #1: WebID Profile document access privileges
This issue is intended to clarify the requirements (and assumptions) of a WebID Profile document as per access privileges. The WebID spec is proposed as an "universal identification mechanism". By definition it is orthogonal to authentication and authorization mechanisms - effectively loosely coupled (as shown in the wild).

There are two places in the current spec that considers the WebID Profile document to be public or protected:

Introduction (non-normative) states:

>WebIDs can be used to build a Web of trust using vocabularies such as FOAF [FOAF] by allowing people to link together their profiles in a **public or protected** manner.

Privacy (non-normative) states:

>A WebID Profile is a **public** document that may contain public as well personal information about the agent identified by the WebID.

Emphasis mine.

While pragmatics may dictate that a WebID Profile document mostly needs to be public, there are ample use cases for it to be protected. Simply put: different profiles may need to be accessible by different parties. An agent may link to their other profiles where they allow access only to their friends, a community or government agents, and so on. The WebID specification does not need to make this call.

I suggest that the spec is updated to clarify by either:
* normatively stating that this spec does not assume or require a WebID Profile document to be public or protected i.e., introduce new text and fix current text.
* non-normatively stating that a WebID Profile document can be public or protected, consistently, i.e., fix the current text.

### PR DESCRIPTION
**Pull request by csarven**
Closes #1

### CODE REVIEW

File: spec/identity/index.html [hunk 1]
```diff
@@ -631,7 +631,7 @@ <h3 id="h3_publishing-a-webid-profile-using-the-rdfa-html-notation"><span class=
 <div class="informative section" id="privacy">
 <h3 id="h3_privacy"><span class="secno">5.4 </span>Privacy</h3><p><em>This section is non-normative.</em></p>
 <p>
-A WebID Profile is a public document that may contain public as well personal information about the agent identified by the WebID.
+A WebID Profile Document may contain public as well personal information about the agent identified by the WebID.
```
Comment (TallTed) [inline spec/identity/index.html:634]: 1. missing "as"
2. "personal" is only applicable to human agents, but WebIDs may also denote software (and other) agents; "private" is therefore a better adjective.
```suggestion
A WebID Profile Document may contain public as well as private information about the agent identified by the WebID.
```

**Review by TallTed — CHANGES_REQUESTED** [review]

**Comment by bblfish** [comment]
Ok, as an editorial change this is an improvement. So it should go through.

**Comment by TallTed** [comment]
I believe @csarven needs to resolve the merge conflicts before it can be merged...

**Comment by csarven** [comment]
Resolved merge conflict.

**Pull request merged**